app/components/press-release/sources-modal.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+import { action } from '@ember/object';
+
+export default class PressReleaseSourcesModalComponent extends Component {
+  @service store;
+
+  @tracked sources = [];
+  @tracked selectedSource;
+
+  constructor() {
+    super(...arguments);
+    this.loadContacts.perform();
+  }
+
+  @task
+  *loadContacts() {
+    this.sources = yield this.store.findAll('contact');
```

Comment:
Op dit moment haal je enkel de eerste pagina van (by default 20, denk ik) contacten op. Als initiele lijst van opties is dit waarschijnlijk ok, best dan wel nog sorteren op naam, maar de search van `PowerSelect` moet een search-operatie naar de backend zijn zodanig dat er niet enkel binnen de contacten van de eerste pagina gezocht wordt.